### ISSUE TITLE
Videos played fine the first day installed but now don't

### ISSUE THREAD

**Issue by darsparx** [meandavejustice/min-vid#1077]
Ok I seem to be having some issues. It was working fine when I installed it a few days ago, but now seems to not work. Any ideas?
(when I say not working I mean, I can send a youtube playlist, or vid to it and it just won't load it no matter what I do)

I also had another probably unrelated issue where I wasn't clicking on it to drag but I nearly lost the thing in one corner....

**Comment by SoftVision-CarmenFat** [comment]
Hi @darsparx
	
Can you please provide us more details such as what Firefox version you have installed, operating system and other add-ons you may have installed (besides Min Vid)?
	
Thanks!

**Comment by darsparx** [comment]
FF is 56.0
OS is windows 10 home 64-bit 1703 15063.332
and the only addons I have are lastpass, minvid, multi-account containers(I need to uninstall that one since I'm not using it), tab groups, and test pilot

**Comment by SoftVision-CarmenFat** [comment]
- Tested this on Firefox 56.0 using Windows 10, x64 and the videos and Playlists are loaded and played correctly.
- @darsparx: Can you please open the browser console and check if any error messages are displayed when videos/Playlists are loading in the mini player?
- Regarding the second issue you mentioned (losing the mini player in one corner), I think you may refer to issue #842, can you confirm that?
Thank you!

**Comment by darsparx** [comment]
Yeah, there are no errors produced from what I can tell, it just sits there and tries to load the video but never does.
